### PR TITLE
Return DoubleEndedIterator from points and points_mut

### LINKED ISSUE #949: geo-types: Ideally, `LineString::coords()` should return `DoubleEndedIterator`
Ideally, `LineString::coords()` should return `DoubleEndedIterator` :)
https://docs.rs/geo-types/latest/src/geo_types/geometry/line_string.rs.html#211

### PR DESCRIPTION
**Pull request by urschrei**
Fixes #949

- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

File: geo-types/CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## unreleased
 
+* Return DoubleEndedIterator from points and points_mut
```
Comment (lnicola) [inline geo-types/CHANGES.md:5]:
```suggestion
* Return `DoubleEndedIterator` from `LineString::points` and `LineString::points_mut`
```

**Comment by urschrei** [comment]
Hmm

**Comment by urschrei** [comment]
bors r=lnicola

**Pull request merged**